docs/customization/overriding_applications.rst

```diff
@@ -90,14 +90,14 @@ Define the application AppConfig
 
 Most of *django-machina*'s applications define sublclasses of Django's ``AppConfig`` which can perform initialization operations. *Django-machina* ``AppConfig`` instances are defined inside sub-modules called ``registry_config``. You need to define an ``AppConfig`` subclass for your custom application by subclassing the overridden application ``AppConfig``. So your application's ``__init__.py`` should report the custom application ``AppConfig``::
 
-    default_app_config = 'apps.forum_conversation.registry_config.ConversationRegistryConfig'
+    default_app_config = 'yourproject.apps.forum_conversation.registry_config.ConversationRegistryConfig'
```

Comment:
In fact the doc stated `apps.forum_conversation.registry_config.ConversationRegistryConfig` because I was assuming that there was an `app` package at the root of the project. Perhaps this was not very clear, though.